packages/cli/src/cli/index.ts

```diff
@@ -1,16 +1,18 @@
 #!/usr/bin/env node
+import { BundleAssetsCommand, BundleCommand, ImportCommand } from '@basemaps/cli-config';
 import { subcommands } from 'cmd-ts';
 
-import { BundleAssetsCommand } from './config/action.bundle.assets.js';
-import { BundleCommand } from './config/action.bundle.js';
-import { ImportCommand } from './config/action.import.js';
-
 export const Cli = subcommands({
   name: 'bmc',
```

Comment:
@tawera-manaena We have subcommands in the `cli` package, not in the `cli-config` package.

Reply:
Good point. @ccbblin We should do that.